### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Viam UFactory xArm Module
 
-This is a [Viam module](https://docs.viam.com/how-tos/create-module/) for [UFactory's](https://www.ufactory.cc/) family of collaborative arms.
+This is a [Viam module](https://docs.viam.com/how-tos/create-module/) for [UFactory's](https://www.ufactory.cc/) X-ARM 6 and X-ARM 7 collaborative arms.
```
Comment (nfranczak) [inline README.md:3]: nit: lite6 is also included in this

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This viam-xarm module is particularly useful in applications that require an xAr
 > [!NOTE]
 > Before configuring your xArm, you must [add a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine).
 
-Navigate to the **CONFIGURE** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click the **+** icon next to your machine part in the left-hand menu and select **Component**. Select the `arm` type, then search for and select the `arm / viam-UFactory-xarm` model. Click **Add module**, then enter a name or use the suggested name for your arm and click **Create**.
+Navigate to the **CONFIGURE** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click the **+** icon next to your machine part in the left-hand menu and select **Component**. Select the `arm` type, then search for and select the `arm / ufactory:xArm6` or `arm / ufactory:xArm7` model, depending on your hardware model. Click **Add module**, then enter a name or use the suggested name for your arm and click **Create**.
```
Comment (nfranczak) [inline README.md:15]: nit: lite6 should also be included

**Review by nfranczak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JessamyT) [reply README.md:3]: Oops yep yep
```suggestion
This is a [Viam module](https://docs.viam.com/how-tos/create-module/) for [UFactory's](https://www.ufactory.cc/) X-ARM 6, X-ARM 7, and LITE 6 collaborative arms.
```

File: README.md (hunk 2 above)
Comment (JessamyT) [reply README.md:15]:
```suggestion
Navigate to the **CONFIGURE** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click the **+** icon next to your machine part in the left-hand menu and select **Component**. Select the `arm` type, then search for and select the `arm / ufactory:xArm6`, `arm / ufactory:xArm7`, or `arm / ufactory:lite6` model, depending on your hardware model. Click **Add module**, then enter a name or use the suggested name for your arm and click **Create**.
```

**Review by nfranczak — APPROVED** [review]
Thank you for handling this! 🚀

**Pull request merged**